cmd/tools/grafana/grafana.go

```diff
@@ -41,9 +42,7 @@ type options struct {
 	addr           string // URL of Grafana server (e.g. "http://localhost:3000")
 	token          string // API token issued by Grafana server
 	dir            string // Directory from which to import dashboards (e.g. "opt/harvest/grafana/dashboards")
-	folder         string // Grafana folder where to upload from where to download dashboards
-	folderId       int64
-	folderUid      string
+	folder         [2]Folder
```

Comment:
would it be clearer if instead of assuming that `folder[0]` is cmode and `[1]` is 7mode to make that explicit with something like:
```go
cmodeFolder Folder
mode7Folder Folder
```

Reply:
done, Thanks.